app/models/nic/interface.rb

```diff
@@ -74,9 +74,11 @@ def identifier_change
       old_value = self.identifier_was || ''
```

Comment:
@ares RHEL7 uses names like the ones in systemd/systemd@e0d4a0ac06afb856c9370c5c256f0f7bb7efdc8e recent Fedora has ones like systemd/systemd@0037a669ac9a2bbedccdb2f483111351e8ff4659

So in recent Fedora `enccw0.0.0600` becomes `enc600` (solving our problem). I also don't really like treating it differently if the name is prefixed with something, but I'm not really sure what to suggest.